sceptre/plan/actions.py

```diff
@@ -534,9 +535,9 @@ def execute_change_set(self, change_set_name):
 
     def list_change_sets(self):
         """
-        Lists the stack's change sets.
+        Lists the Stack's Change Sets.
 
-        :returns: The stack's change sets.
+        :returns: The Stack's Change Sets.
         :rtype: dict
```

Comment:
Whilst this is accurate for the actual boto call, the `except` will return a list (which will probably cause problems elsewhere).